src/ProjectTemplates/test/SpaTemplateTest/SpaTemplateTestBase.cs

```diff
@@ -254,7 +254,8 @@ private void TestBasicNavigation(bool visitFetchData, bool usesAuth, IWebDriver
 
                 badEntries = badEntries.Where(e =>
                     !e.Message.Contains("failed: WebSocket is closed before the connection is established.")
-                    && !e.Message.Contains("[WDS] Disconnected!"));
+                    && !e.Message.Contains("[WDS] Disconnected!")
+                    && !e.Message.Contains("Timed out connecting to Chrome, retrying"));
```

Comment:
Isn't this what selenium throws when it can't connect to the instance? Should this be in this category?

Reply:
A bit of context will help here.

Reply:
This is what selenium throws if it didn't connect to the instance on the first try. It does multiple tries, and if there is a total failure to connect it will give a different message. This started happening recently on my dev-box, everything still worked as per the checks above, so the error seems to just be about how quickly your instance spins up. Since that's not what we're testing here I added this exception to increase reliability of these tests.